src/index.html

```diff
@@ -7,6 +7,68 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-fri calendar--month-length-30">
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
```

Comment:
```suggestion
      <div class="calendar-day"></div>
      <div class="calendar-day"></div>
      <div class="calendar-day"></div>
```

Add blank line only between multiple sibling blocks of html like in example below:

```
<li>
  <a>first link</a>
</li>

<li>
  <a>second link</a>
</li>
```